### ISSUE TITLE
--sample flag question 

### ISSUE THREAD

**Issue by NathanWalker** [nstudio/xplat#19]
From twitter:
> Hi. Was trying to test out xplat with --sample. But looks like items not added to xplat/nativescript/features. "Please add a @NgModule annotation.
../../xplat/nativescript/features/index.ts(1,15): error TS2307: Cannot find module './items'." Ideas? Thnx!

**Comment by NathanWalker** [comment]
When using the `--sample1` flag you should also generate xplat with the `--sample` flag - search for 'Generate with a sample feature' section here: https://nstudio.io/xplat/

**Comment by bubbleheadinc** [comment]
@NathanWalker Thanks for the quick reply. I did follow the 'Generate with sample feature' docs. Not sure what I'm missing, but it seems the Items feature isn't being generated inside xplat tree. 

These are the steps I took:

```
create-nx-workspace testxplatws  

cd testxplatws

npm install @nstudio/schematics --save-dev

ng g xplat --prefix=foo --sample

ng g app.nativescript yourapp --sample

npm i

npm run start.nativescript.yourapp.ios
```

And I get these errors:

```
ERROR in : Unexpected value 'ItemsModule in /Users/sam/dev/testxplatws/xplat/nativescript/index.ts' imported by the module 'ItemsModuleLoader in /Users/sam/dev/testxplatws/apps/nativescript-yourapp/app/features/items/items.loader.module.ts'. Please add a @NgModule annotation.
../../xplat/nativescript/features/index.ts(1,15): error TS2307: Cannot find module './items'.
app/features/items/items.loader.module.ts(9,10): error TS2305: Module '"/Users/sam/dev/testxplatws/xplat/nativescript/index"' has no exported member 'ItemsModule'.
app/features/items/items.loader.module.ts(9,23): error TS2305: Module '"/Users/sam/dev/testxplatws/xplat/nativescript/index"' has no exported member 'ItemsComponent'.
app/features/items/items.loader.module.ts(9,39): error TS2305: Module '"/Users/sam/dev/testxplatws/xplat/nativescript/index"' has no exported member 'ItemDetailComponent'.
```
![image](https://user-images.githubusercontent.com/4236279/44160261-e18ddd00-a087-11e8-8f8a-3e05414143b1.png)

**Comment by NathanWalker** [comment]
Ah I see so likely could be 2 things:
* could be a bug
* could be just missing an option in the docs, try this:

```
ng g app.nativescript yourapp --sample --routing
```

Thanks for reporting this in either case. It looks like the route config may be expected to be there.

**Comment by NathanWalker** [comment]
It looks like items weren't generated with your xplat structure so it's possible the `--sample` flag when `ng g xplat` needs an adjustment. I'll label as bug and could be a recent regression.

**Comment by bubbleheadinc** [comment]
Kewl thanks. I just tried with the --routing flag with the same results.

**Comment by NathanWalker** [comment]
@bubbleheadinc this is now fixed in `6.2.2` of the schematics. If you update and then generate again as per the guide it will work as expected 👍 

https://github.com/nstudio/xplat/pull/21
